src/data/items/slotNeck.js

```diff
@@ -192,6 +192,25 @@ export default {
 		description: "+1 Chemistry Upgrade",
 		providesUpgrade: "chemDispenser"
 	},
+	capeResearch: {

```

Comment:
You don't have an upgrade tied to this.
description: "Years of being bullied pay off" or some other fun flavor text.

Reply:
A temporary sixth R&D upgrade wouldn't work very well since there isn't anything that needs >1000 research points to get, plus I don't know how it would work if you had >1000 points then took off the cap and lowered your max capacity to below your total (I imagine it would be buggy), so I didn't want to touch it